plugin/c.go

```diff
@@ -117,4 +117,24 @@ package plugin // import "collectd.org/plugin"
 //   }
 //   return (*register_log_ptr) (name, callback, user_data);
 // }
+//
+// int (*register_complex_config_ptr) (const char *, plugin_complex_config_cb);
```

Comment:
This function pointer should be `static`, i.e.:

```c
static int (*register_complex_config_ptr) (const char *, plugin_complex_config_cb);
```

(Yes, I know that this is not (yet) consistently done for other callbacks.)

Reply:
Other function pointers will be fixed in #64.